src/github/client/github-installation-client.ts

```diff
@@ -362,6 +364,20 @@ export class GitHubInstallationClient extends GitHubClient {
 		return response?.data?.data?.viewer?.repositories?.totalCount;
 	}
 
+	public async getDependabotAlertsPage(owner: string, repoName: string, perPage = 1, cursor?: string): Promise<getDependabotAlertsResponse> {
```

Comment:
Why `perPage` is 1? Wouldn't that be super slow?

Reply:
It is the default value, same value used in getBranches as well. Updated to 20 now.

Reply:
This is how actual page size calculated
```
const nPages = Math.min(
				// "|| 20" is purely to simplify testing and avoid mocking it all the time
				await numberFlag(NumberFlags.BACKFILL_PAGE_SIZE, 20, jiraHost) || 20,
				// The majority of GitHub APIs hard-limit the page size to 100
				100
			);
```